Jenkinsfile.release

```diff
@@ -14,7 +14,7 @@ pipeline {
   }
 
   parameters {
-    booleanParam(defaultValue: false, description: 'Should the build be uploaded to S3?', name: 'SHOULD_UPLOAD_BUILD')
+    booleanParam(defaultValue: false, description: 'Should run the Authorization?', name: 'SHOULD_AUTHORIZE')
```

Comment:
Now, every time a PR is merged into master we want to automatically create a build tar and bump the nightly DC/OS PR. So from "Should upload", we can change this to "Should Authorize". 

The Authorize requires someone to manually go into Jenkins and click on Proceed which will make the whole CI/CD process much less automatic and streamlined I think. But we can keep this if we want to enable it in the future or from Jenkins.

Reply:
The reason why it asks for authorization currently is because you don't run it on the master branch. Once this is merged it will no longer ask for authorization as this jenkins job normally only runs on master.

Reply:
The switch was thought as "should this be a real release that is used somewhere in production"